### PR TITLE
Feature: adds a php-fpm 7.3 container

### PR DESCRIPTION
**Pull request by lsh-0**
* OS upgrade from `jessie` to `stretch`

todo:

* gnupg and apt-key [are no longer supported](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=869873) and a replacement will need to be found
* php 7.3 configuration required

cc @giorgiosironi

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Not sure whether we should drop the base images (as they are one additional step in the chain of updates) or whether they are worth keeping? They do create users and install basic things that is useful for existing PHP applications (that weren't Dockerized from the start).

**Comment by lsh-0** [comment]
I mentioned in another ticket, about the data-pipeline project I think, about harmonising the base image of the containers in use there, because they can get quite large - 700MB to over a GB - and we have to put them on the /ext drive. If the containers shared a base image, that would reduce size but be less flexible for those devs who want the bleeding edge language features.

I'm skeptical about just *how much* flexibility is needed and what OS constraints are unreasonable to impose by us on them. These images derive from Debian Jessie and Stretch and they are already quite strict... this apt-key thing is just an example of that.

At the end of the day though, a typical machine will have 2 or 3 containers on them - the app, a webserver, and a db, maybe, and all of them sharing the same base image (hopefully). And this won't be for very long if I understand your EKS work right - there will be pools of ec2 instances running mixed containers.

Regarding ease of rebuilding, can we 'flatten' these containers? Concatenate them into a single Dockerfile and build that?

**Comment by giorgiosironi** [comment]
`gnupg` is not installed by default but can be installed preemptively.

**Comment by giorgiosironi** [comment]
Counter-review?

File: Dockerfile.php_7.3_fpm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM php:7.3.4-fpm-stretch
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+    libfcgi0ldbl \
+    libfcgi-bin \
+    gnupg \
```
Comment (thewilkybarkid) [inline Dockerfile.php_7.3_fpm]: What are these 2 for?

Comment (giorgiosironi) [reply Dockerfile.php_7.3_fpm]: The other one is `libfcgi-bin` which has been split from `libfcgi0ldbl` to contain the binary utility for health checking a FastCGI server.

Comment (thewilkybarkid) [reply Dockerfile.php_7.3_fpm]: Need to (explicitly) install both of them?

Comment (giorgiosironi) [reply Dockerfile.php_7.3_fpm]: Possibly not

**Pull request merged**